src/main/java/com/github/sdorra/webresources/WebResources.java

```diff
@@ -143,8 +143,8 @@ static String etag(Path path) throws IOException {
         return etag(name, size, lastModified);
     }
 
-    private static String etag(String name, long size, long lastModifid) {
-        return String.format("%s_%s_%s", name, size, lastModifid);
+    private static String etag(String name, long size, long lastModified) {
+        return "W/\"" + String.format("%s_%s_%s", name, size, lastModified) + "\"";
```

Comment:
Why two steps, first `String.format` and then concatenation. Why not use `String.format`, only?